oauth/oauth.ts

```diff
@@ -0,0 +1,19 @@
+import { User } from "../models/Model";
+
+const passport = require("passport");
+const GoogleStrategy = require("passport-google-oauth").OAuth2Strategy;
+
+passport.use(
+  new GoogleStrategy(
+    {
+      clientID: "API key 1",
+      clientSecret: "AIzaSyDxHJy8GFT5q9shMJaRzmqfjNXbQmkCThE",
```

Comment:
This will need pulled out and put into a .env file

Reply:
Yeah was having weird things happening since I didn't push my changes the day before and some things were changed causing some conflicts. This has been moved to .env and changed as it was the wrong information we needed I believe.